pkg/agent/core/ngt/handler/grpc/linear_search.go

```diff
@@ -86,6 +86,18 @@ func (s *server) LinearSearch(ctx context.Context, req *payload.Search_Request)
 				})
 			log.Debug(err)
 			attrs = trace.StatusCodeAborted(err.Error())
+		case errors.Is(err, errors.ErrFlushingIsInProgress):
+			err = status.WrapWithAborted("Search API aborted to process search request due to flushing indices is in progress", err,
+				&errdetails.RequestInfo{
+					RequestId:   req.GetConfig().GetRequestId(),
+					ServingData: errdetails.Serialize(req),
+				},
+				&errdetails.ResourceInfo{
+					ResourceType: ngtResourceType + "/ngt.Search",
+					ResourceName: fmt.Sprintf("%s: %s(%s)", apiName, s.name, s.ip),
+				})
+			log.Debug(err)
+			attrs = trace.StatusCodeAborted(err.Error())
```

Comment:
you have to update the search API doc cuz it is a new status code.
ref: https://vald.vdaas.org/docs/api/search/#status-code-6

Also, this document has to be updated.
https://vald.vdaas.org/docs/api/status/